include/dev/MAX31855.hpp

```diff
@@ -22,12 +23,31 @@ class MAX31855 {
      */
     explicit MAX31855(SPI& spi, uint8_t device);
 
+    /*
+     * The status of the MAX31855 after starting a temperature read.
+     *
+     * NO_ERROR: No error
+     * SCV_FAULT: Short to VCC
+     * SCG_FAULT: Short to GND
+     * OC_FAULT: Open Circuit
+     * MAX31855_ERROR: Error reading from the MAX31855
+     */
+    enum class MaxStatus {
+        NO_ERROR = 0,
```

Comment:
This is how we normally handle this
```suggestion
        OK = 0,
```